authentication/python/azure/README.md

```diff
@@ -0,0 +1,17 @@
+# To-do
+
+## Using the library
+
+# To-do
+
+![Python](https://github.com/microsoft/kiota/actions/workflows/authentication-python-azure.yml/badge.svg)
+
+- [] coverage code
+- [x] analyzers
+- [] unit test project
+- [x] docs comments
+
+## Using the Azure Authentication Library
+
+1. Install the package by running the command
+`pip install -i https://test.pypi.org/simple/ kiota-authentication-azure`
```

Comment:
```suggestion
`pip install -i https://test.pypi.org/simple/kiota-authentication-azure`
```
There's a space character which would cause issues if someone copy pasted this.